src/HistorySaver.java

```diff
@@ -0,0 +1,20 @@
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+
+public class HistorySaver {
+
+    public void saveHistory( String userName, String date, String service, String destination, String amount, String currency) throws IOException {
```

Comment:
try to write in multiple lanes, it is hard to read
ex:
public void saveHistory( String userName, String date, String service, String destination, String amount, 
String currency) throws IOException {